### ISSUE TITLE
Sort by quality in manual search

### ISSUE THREAD

**Issue by ngosang** [pymedusa/Medusa#5217]
**Describe the bug**
Sort by quality column in manual search is not working as expected.

**To Reproduce**
1. Go to manual search
2. Sort by quality column

**Expected behavior**
Order from lower to higher quality (or reverse)

**Screenshots**
![image](https://user-images.githubusercontent.com/10577978/45574828-937c1d00-b871-11e8-893f-5d6cddd1f37a.png)

**Medusa (please complete the following information):**
 - Branch: **develop**

**Comment by rafi-d** [comment]
+1 !
Also, once you try to sort them, there is no going back to the default state, which is sorted correctly...

**Comment by rafi-d** [comment]
Is this been fixed in 0.3.2?

**Comment by p0psicles** [comment]
No. We first need to vueify displayShow. That's almost finished. After that snatchSelection comes
